src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheckTest.java

```diff
@@ -741,16 +737,45 @@ public void testNoPackage2() throws Exception {
 
         createChecker(checkConfig);
         final String[] expected = {
-            "3: " + getCheckMessage(MSG_LINE_SEPARATOR,
-                "com.puppycrawl.tools.checkstyle.utils.AnnotationUtil.containsAnnotation"),
             "7: " + getCheckMessage(MSG_LINE_SEPARATOR,
                 "com.sun.accessibility.internal.resources.*"),
-            "11: " + getCheckMessage(MSG_LINE_SEPARATOR, "java.util.Arrays"),
-            "19: " + getCheckMessage(MSG_LINE_SEPARATOR,
-                "org.apache.commons.beanutils.converters.ArrayConverter"),
         };
         verify(checkConfig, getNonCompilablePath("InputCustomImportOrderNoPackage2.java"),
             expected);
     }
 
+    @Test
+    public void testInputCustomImportOrderMultipleSeparators() throws Exception {
```

Comment:
@romani , IIUC this new test case should satisfy as you mentioned,

> test case(minization is requied) to add to Inputs:
https://romani.github.io/issue-6680-pr-6830/diff/apache-ant/index.html#A57, https://romani.github.io/issue-6680-pr-6830/diff/apache-ant/index.html#A58

for 
> it will be good to copy whole file to inputs(no package, few multiline separators)

A dedicated input, `InputCustomImportOrderMultipleSeparators.java`, was added

Reply:
> InputCustomImportOrderMultipleSeparators.java 

this file have "package com.puppycrawl.too....."
I requested creation of Input file with same imports but withtout definition of package (no package at all). See source file at https://romani.github.io/issue-6680-pr-6830/diff/checkstyle/index.html#A3 .
New Input file should be placed at https://github.com/checkstyle/checkstyle/tree/master/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder

Reply:
Got it !